README.md

```diff
@@ -9,7 +9,17 @@ We'd love to see you contributing to our projects, please just follow a few conv
 - No trailing whitespace. Blank lines should not have any spaces.
 - Do not indent after private/protected.
 - Use Ruby >= 1.9 syntax for hashes. Prefer `{ a: :b }` over `{ :a => :b }`.
-- Prefer `&&`/`||` over and/or.
+- Prefer `&&`/`||` over `and`/`or`.
+- Use `and`/`or` only for the early returns:
+``` ruby
+def some_method_with_and
+  head 401 and return
+end
+
+def some_method_with_or(user)
+  user.admin? or raise UnauthorizedRequestError
```

Comment:
@Mandaryn Well, yeah, that's why it should be used for early returns only.

It's not objectively better than the code you provided, it's just a matter of taste. For me `or` looks much better, maybe because I find `unless` quite unreadable, but even if it was `raise UnauthorizedRequestError if !user.admin?` I would still prefer `or`